plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/ReflectionUtils.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2019 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ide.server;
+
+import java.lang.reflect.Field;
+
+/**
+ *
+ */
+public class ReflectionUtils {
```

Comment:
Should be moved to bundle `org.eclipse.n4js.utils`.

Reply:
Ok. Alternatively `ReflectionUtils` will be removed.